src/components/EditCoursePage/CollapsibleCourseRunFields.jsx

```diff
@@ -80,7 +81,7 @@ export const BaseCollapsibleCourseRunFields = ({
           type="date"
           component={RenderInputTextField}
           format={value => getDateString(value)}
-          label={<FieldLabel text="Start date" required />}
+          label={<FieldLabel text="Start date" required requiredForSubmit />}
```

Comment:
In my mind it's nice to have both in case `required` is one day removed but it's still `requiredForSubmit`... but happy to remove it if there is disagreement

Reply:
Naw, I think that's a fair call.